forui/lib/src/widgets/breadcrumb.dart

```diff
@@ -0,0 +1,284 @@
+final class FBreadcrumb extends StatelessWidget {
+  /// The breadcrumb's style. Defaults to the appropriate style in [FThemeData.breadcrumbStyle].
+  final FBreadcrumbStyle? style;
+
+  /// The list of breadcrumb items.
+  ///
+  /// Each item in the list is typically an [FBreadcrumbItem], with a [divider] placed between consecutive items.
+  /// The last item in the list is usually an [FBreadcrumbItem] with the `selected` property set to true,
+  /// to indicate the current page in the breadcrumb.
+  final List<Widget> children;
+
+  /// The divider icons place in between the children.
+  ///
+  /// Defaults to an `FAssets.icons.chevronRight` icon.
+  final Widget? divider;
+
+  /// Creates an [FBreadcrumb].
+  const FBreadcrumb({
+    required this.children,
+    this.style,
+    this.divider,
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final style = this.style ?? context.theme.breadcrumbStyle;
+    final divider = this.divider != null
+        ? FIconStyleData(
+            style: style.iconStyle,
+            child: this.divider!,
+          )
+        : FIcon(
+            FAssets.icons.chevronRight,
+            color: style.iconStyle.color,
+            size: style.iconStyle.size,
+          );
+
+    return Row(
+      children: [
+        for (final (index, item) in children.indexed) ...[
+          FBreadcrumbItemData(style: style, child: item),
+          if (index < children.length - 1) Padding(padding: const EdgeInsets.symmetric(horizontal: 5), child: divider),
+        ],
+      ],
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty('style', style));
+  }
+}
+
+/// The [FBreadcrumbItem] data.
+class FBreadcrumbItemData extends InheritedWidget {
+  /// Returns the [FBreadcrumbItemData] of the [FBreadcrumb] in the given [context].
+  ///
+  /// ## Contract
+  /// Throws [AssertionError] if there is no ancestor [FBreadcrumb] in the given [context].
+  @useResult
+  static FBreadcrumbItemData of(BuildContext context) {
+    final data = context.dependOnInheritedWidgetOfExactType<FBreadcrumbItemData>();
+    assert(data != null, 'No FBreadcrumbData found in context');
+    return data!;
+  }
+
+  /// The breadcrumb's style.
+  final FBreadcrumbStyle style;
+
+  /// Creates a [FBreadcrumbItemData].
+  const FBreadcrumbItemData({
+    required this.style,
+    required super.child,
+    super.key,
+  });
+
+  @override
+  bool updateShouldNotify(FBreadcrumbItemData oldWidget) => style != oldWidget.style;
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty('style', style));
+  }
+}
+
+/// A breadcrumb item.
+mixin FBreadcrumbItem on Widget {
+  /// Creates a basic crumb.
+  ///
+  /// It is typically used to represent a single item in the breadcrumb.
```

Comment:
I'm sorry but this documentation just isn't acceptable. It doesn't tell us anything meaningful. It needs to be much more descriptive. 

What's a "basic" crumb? 

The two sentences should be combined in the form, "Creates a crumb that typically represents ...". Perhaps a page in the navigation history?